canu/generate/switch/config/config.py

```diff
@@ -1097,6 +1103,8 @@ def groupby_vlan_range(vlan_list):
     Returns:
         list of vlans formatted
     """
+    if None in vlan_list:
```

Comment:
Do you really want to throw out the entire list because of a single entry of "None"?  Wouldn't we just want to prune the list?  Not quite sure what you're doing here.  Also the return value is nothing and the docs for this function say returning a list of vlans.

Reply:
Function now prunes out "None"